releases/release-1.18/release-notes-draft.md

```diff
@@ -1,163 +1,243 @@
-# v1.18.0-rc.1
+## Changelog since v1.17.0
 
-## Changelog since v1.17.0-rc.1
+A complete changelog for the release notes is now hosted in a customizable
+format at [https://relnotes.k8s.io][1]. Check it out and please give us your
+feedback!
+
+[1]: https://relnotes.k8s.io/?releaseVersions=1.18.0
+
+## What’s New (Major Themes)
+
+
+
```

Comment:
nitty nit: I'd prefer to avoid such multiple newlines, one newline should be enough

Reply:
Removed extraneous whitespacing throughout